src/components/AddWeddingForm/AddWeddingForm.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import './AddWeddingForm.css'
+import empty from '../../assets/emptyGuestList.png'
+import { postAWedding } from '../../apiCalls'
+import { StyledButton } from '../App/styledComponents.styles'
+
+
+type NewWedding = {
```

Comment:
Refactor opportunity: export this to the dashboard so we don't set type for Wedding twice